src/runtime/components/forms/Select.vue

```diff
@@ -319,3 +319,13 @@ export default defineComponent({
   }
 })
 </script>
+<style scoped>
+.form-select {
```

Comment:
Why do you need to add this?

The `form-select` class already does this:

![CleanShot 2023-07-26 at 18.32.17@2x.png](https://volta.s3.fr-par.scw.cloud/Clean_Shot_2023_07_26_at_18_32_17_2x_1a37a1d924.png)

https://github.com/tailwindlabs/tailwindcss-forms/blob/master/src/index.js#L37

Reply:
Oh OK.